src/sentry/api/serializers/models/release_threshold.py

```diff
@@ -0,0 +1,20 @@
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models.release_threshold.constants import (
```

Comment:
Maybe `utils` is more appropriate 🤷 
I dunno haha. 

Just a thought - i think constants is probably fine tho